capstone-backend/src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -212,8 +213,14 @@ static VolumeData individualBookToVolumeData(JsonElement singleBook) {
     String canonicalVolumeLink = canonVolumeElement != null ? canonVolumeElement.getAsString() : "";
 
     JsonElement thumbnailElement = volumeInfoObj.get("imageLinks");
-    String thumbnailLink = thumbnailElement != null ? thumbnailElement.getAsJsonObject().get("thumbnail").getAsString()
-        : "";
+    ArrayList<String> sizes = new ArrayList<>(Arrays.asList("extraLarge", "large", "medium", "small", "thumbnail"));
```

Comment:
Why is the last size "thumbnail"?

Reply:
`thumbnail` is the largest default image given by the API, you can see in this example here: https://www.googleapis.com/books/v1/volumes/x0BHx72wj1sC